src/research-grant.md

```diff
@@ -0,0 +1,22 @@
+---
+permalink: /research-grant/
```

Comment:
We tend to keep URL slugs brief (`/blog`, `/docs`, `/logo`, `/features`, etc.) so how about just `/grant/`? Just a suggestion, not a blocker, feel free to do whatever you want :)

Reply:
That URL is what @hannespayer asked for, but if he agrees, I don't mind changing to shorter one as well :)

Reply:
Good point regarding brief URLs. /grant sounds good. Maybe we should add /research at some point to advertise our research investment? Different topic...